HiP-UserStore.Model/Rest/UserQueryArgs.cs

```diff
@@ -11,5 +13,11 @@ public class UserQueryArgs : QueryArgs
         /// Restricts the response to users having an email that begins with the specified string
         /// </summary>
         public string EmailBeginning { get; set; }
+                
+        /// <summary>
+        /// If set, only those users, with their IDs specified, are included in the response.
+        /// If null, all IDs are included in the response.
+        /// </summary>
+        public List<string> UsersIdsToBeIncluded { get; set; }
```

Comment:
To be consistent to the DataStore API, the parameter should be called includeOnly (like [here](https://github.com/HiP-App/HiP-DataStore/blob/develop/HiP-DataStore.Model/Rest/QueryArgs.cs)). The comment looks good and can stay